scene/gui/rich_text_label.cpp

```diff
@@ -1793,6 +1794,17 @@ void RichTextLabel::_notification(int p_what) {
 		case NOTIFICATION_TRANSLATION_CHANGED: {
 			_stop_thread();
 			main->first_invalid_line.store(0); //invalidate ALL
```

Comment:
These lines are no longer necessary, `clear` and `parse_bbcode` will do it automatically.